lib/cli_opts.py

```diff
@@ -1204,7 +1204,7 @@ class CliOption(Option):
 
 TIMEOUT_OPT = cli_option("--timeout", dest="timeout", type="int",
                          default=constants.DEFAULT_SHUTDOWN_TIMEOUT,
-                         help="Maximum time to wait")
+                         help="Maximum number of minutes to wait")
```

Comment:
Not sure "number of minutes" sounds idiomatic English.

Maybe reword into "How much time (in minutes) to wait"? Or something like this. Same below.